pyorbital/geoloc_avhrr.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2025 PyTroll Community
+
+# Author(s):
+
+#   Martin Raspaud <martin.raspaud@smhi.se>
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""This module provides geoloc operations specific to the avhrr instrument.
+
+In particular, it provides functions that allow matching gcp location in swath coordinates to reference positions, and
+then minimise the distance to these positions by adjusting the time offset and attitude.
+"""
+
+import logging
+
+import numpy as np
+from pyproj import Geod
+
+from pyorbital.geoloc import ScanGeometry, compute_pixels, get_lonlatalt
+
+logger = logging.getLogger(__name__)
+geod = Geod(ellps="WGS84")
+
+def compute_avhrr_gcps_lonlatalt(gcps, max_scan_angle, rpy, start_time, tle) -> None:
+    """Compute the longitute, latitude and altitude of given gcps (scanlines, columns of the swath).
+
+    The gcps are arbitrary location in swath coordinates, for example (10.3, 7.7) for a gcp at line 10.3 in the swath,
+    and column 7.7. This function is the returning the geographical coordinates of the gcps.
```

Comment:
```suggestion
    and column 7.7. This function returns the interpolated geographical coordinates of these gcps.
```

Reply:
It's not interpolated actually, but I'll fix the wording.